foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -31,9 +31,11 @@ sparkApp:
         #if (${enablePySparkSupport})
         - https://repo1.maven.org/maven2/org/apache/sedona/sedona-python-adapter-3.0_2.12/${versionSedona}/sedona-python-adapter-3.0_2.12-${versionSedona}.jar
         - https://repo1.maven.org/maven2/org/datasyslab/geotools-wrapper/${versionGeotools}/geotools-wrapper-${versionGeotools}.jar
-        - https://repo1.maven.org/maven2/org/postgresql/postgresql/${versionPostgresql}/postgresql-${versionPostgresql}.jar
         #end
         #end
+        #if (${enablePySparkSupport})
```

Comment:
S: What do we gain by pulling this out to here?

Reply:
I was just trying to be consistent with the usePackageDeps conditional based on the changes below. That said, depending on the outcome of the other comment in this file below, it may not be warranted at all.

Reply:
It looks to me like we're saying that _if PySpark Support is enabled_, we always want to download the Postgres jar directly, whether we're using package deps or not.  It looks to me like for pyspark pipelines, we would have both the Jar and Package dependency simultaneously.

Reply:
It seems like it's configured to pull one or the other.

```
      #if (!${usePackageDeps})
      jars:
      ...
      #else
      packages:
      ...
      #end

```

Reply:
Ah, I miscounted my #end calls.  Like I said separately, though, this still feels aggressively like the wrong condition.

Reply:
Adressed, same as comment below